audiobookshelf/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+volumes:
+  audiobooks:
+  podcasts:
+  config:
+  metadata:
+    
+services:
+  audiobookshelf:
+    image: ghcr.io/advplyr/audiobookshelf:latest
```

Comment:
we shouldn't use `latest`, but instead it should have an AUDIOBOOKSHELF_VERSION variable in the config.